apps/projects/app/components/Shared/FilterBar/FilterBar.js

```diff
@@ -280,14 +323,14 @@ const FilterBar = ({
             {filtersData.statuses[status].count})
           </ActionLabel>
         </FilterMenuItem>
-      )),
+      ))}
       <hr css={`
         height: 1px;
         border: 0;
         width: 100%;
         background: ${theme.border};
       `} />,
```

Comment:
I think we need to get rid of this comma, now.

<img alt="" src="https://user-images.githubusercontent.com/221614/67348419-e1526280-f512-11e9-89d3-ad647383f218.png" width="250px">